tests/integration/examples/01_create_reqif_objects/script.py

```diff
@@ -32,7 +32,7 @@ class ExampleIdentifiers:
     # For example, 550e8400-e29b-41d4-a716-446655440000 or similar.
     # For this example, we keep it as strings.
     STRING_DATATYPE_ID = create_uuid()
-    SPEC_ATTRIBUTE_ID = create_uuid()
+    SPEC_ATTRIBUTE_IDs = list(map(create_uuid, range(9)))
```

Comment:
This change seems unrelated to the topic of types. Could you explain your intention here?

Reply:
Oh wait, nevermind. It is clear now :smile: